src/app/shared/ai-bubble/ai-bubble.component.html

```diff
@@ -0,0 +1,25 @@
+<a
+  *ngIf="isPublic"
+  [href]="Dockstore.DOCUMENTATION_URL + '/faq.html'"
+  target="_blank"
+  rel="noopener noreferrer"
+  data-cy="ai-bubble"
+  class="bubble ai-bubble"
+  matTooltip="This topic sentence was AI generated."
```

Comment:
Both `<a>` elements are identical except for the tooltip. A simplification would be to remove the `ngIf` and put a conditional inside the `matTooltip`.